ui/dgt/src/play.ts

```diff
@@ -775,67 +775,76 @@ export default function (token: string) {
         }
         else {
           //A move was received
-          SANMove = String(message.param.san[message.param.san.length - 1]).trim();
-          if (verbose) console.info('onmessage - SANMove = ' + SANMove);
-          //Store the postition too to help prevent duplicates
+          //Get all the moves on the param.san that are not present on lastLegalParam.san
+          //it is possible to receive two new moves on the message. Don't assume only the last move is pending.
+          var movesToProcess = 1;
```

Comment:
Nowadays we pretty much always want to use `let` rather than `var`. Or of course `const` wherever possible.